### PR TITLE
[MB-2612] Add dimensions for crating to lookups

### PR DESCRIPTION
**Pull request by jacquelineIO**
## Description

This adds length, width, and height dimensions for lookups and to the database to crating/uncrating service items.

## Reviewer Notes

n/a

## Setup

Just runs normal `make server_test`

## Code Review Verification Steps

* [ ] The requirements listed in [Querying the Database Safely](https://github.com/transcom/mymove/wiki/Backend-Programming-Guide#querying-the-database-safely) have been satisfied.
* Any new migrations/schema changes:
  * [ ] Follow our guidelines for zero-downtime deploys (see [Zero-Downtime Deploys](https://github.com/transcom/mymove/wiki/migrate-the-database#zero-downtime-migrations))
  * [ ] Have been communicated to #g-database
* [ ] Request review from a member of a different team.
* [ ] Have the Jira acceptance criteria been met for this change?

## References

* [Jira story](https://dp3.atlassian.net/browse/MB-2612) for this change

## Screenshots
n/a

### CODE REVIEW

File: pkg/unit/thousandth_inches.go [hunk 1]
```diff
@@ -8,3 +13,17 @@ func (t ThousandthInches) Int32Ptr() *int32 {
...
+
+// ToFeet returns feet for this value
+func (t ThousandthInches) ToFeet() float64 {
+	feet := float64(t) / thousandthInchPerFoot
+
+	return feet
+}
+
+// ToInches returns inches for this value
+func (t ThousandthInches) ToInches() float64 {
```
Comment (reggieriser) [inline pkg/unit/thousandth_inches.go:25]: I like these new utility functions ... should we add some simple tests for them?  Looks like we do already have a test for the one conversion function that was already here.

Comment (jacquelineIO) [reply pkg/unit/thousandth_inches.go:25]: @reggieriser that makes fence, I was on the fence and was testing it in the other test, but makes sense to have it's own too.

File: migrations/app/schema/20210726214608_add_dimension_param_keys.up.sql [hunk 2]
```diff
@@ -0,0 +1,26 @@
+------ Add new service item param keys ------
+INSERT INTO service_item_param_keys
+(id, key,description,type,origin,created_at,updated_at)
+VALUES
+('9df726f8-5ae7-4749-8446-28815ff16271', 'DimensionLength', 'Compounded escalation factor applied to final price', 'INTEGER', 'PRIME', now(), now()),
+('79bdb87e-776b-43fe-a54b-40e519a9a76f', 'DimensionWidth', 'Price, rate, or factor used in calculation', 'INTEGER', 'PRIME', now(), now()),
+('33d99169-b36c-42f8-ad81-99cb54c9a723', 'DimensionHeight', 'True if this is a peak season move', 'INTEGER', 'PRIME', now(), now());
+
```
Comment (LeDeep) [inline migrations/app/schema/20210726214608_add_dimension_param_keys.up.sql:8]: these descriptions don't seem accurate. copy/paste, maybe?

**Review by LeDeep — CHANGES_REQUESTED** [review]

**Review by gidjin — APPROVED** [review]
Changes look good to me. I'm good with this going in pending fixing the comments that Deep pointed out.

File: migrations/app/schema/20210726214608_add_dimension_param_keys.up.sql (hunk 2 above)
Comment (jacquelineIO) [reply migrations/app/schema/20210726214608_add_dimension_param_keys.up.sql:8]: @LeDeep oops good catch, thank you.

**Review by LeDeep — APPROVED** [review]
database changes lgtm! thanks for making the changes.

**Pull request merged**